### PR TITLE
Judicial System: Moved court end time to the confirmation screen (#2307)

### PR DESCRIPTION
**Pull request by gudjong**
Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>

# Move Court End Time

https://app.asana.com/0/1199153462262248/1199353523039760

## What

☝️ 

## Why

Minor release

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.spec.tsx [hunk 1]
```diff
@@ -1,17 +1,73 @@
...
+
+    // Act
+    render(
+      <MockedProvider
+        mocks={[
+          ...mockCaseQueries,
+          ...mockJudgeQuery,
+          ...mockUpdateCaseMutation([
+            {
+              courtStartTime: '2020-09-16T15:55:000Z',
```
Comment (oddsson) [inline apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.spec.tsx:31]: Hmm, this is not correct 🤔

This should be `courtEndTime`. Could you please see if the test runs like that?

**Review by oddsson — CHANGES_REQUESTED** [review]
One thing regarding a test.. I'll look into it

File: apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.spec.tsx (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.spec.tsx]: The test also succeeds with courtEndTime set and I have pushed that change. Note however that this change will not be merged to the main branch, so if it is also wrong there, then it needs to be fixed there as well.

**Review by pshomov — APPROVED** [review]
LGTM

**Review by oddsson — APPROVED** [review]

**Pull request merged**